projects/mysadna/src/app/components/index.ts

```diff
@@ -0,0 +1,16 @@
+export * from './project-details';
```

Comment:
Did you hear about `index` files?
It prevents using long file names. Also groups children in parent folder.

Reply:
> asterix
It means **all**

> both export and import
Good question!
We need **export** to be able to import elements directly from components folder. E.g.:
```
import { ProjectsComponent, UsersComponent } from '@/components';
```
In that way we don't need to set direct path. Index file does all the work. We even can import several components in on string.

We need **import**, to import components inside index file and then put them to ComponentList, which we export too. ComponentList can be imported to other files. Usually we need the lists, such as ComponentList, to add it to module, or to another index file.